pkgs/development/tools/misc/chrpath/default.nix

```diff
@@ -17,7 +21,7 @@ stdenv.mkDerivation rec {
       binary. The rpath, or runpath if it is present, is where the runtime
       linker should look for the libraries needed for a program.
     '';
-    homepage = "https://tracker.debian.org/pkg/chrpath";
+    homepage = "https://codeberg.org/pere/chrpath";
     license = licenses.gpl2;
```

Comment:
`licenses.gpl2` is deprecated, use `gpl2Only` or `gpl2Plus` as appropriate.

Reply:
I'm not sure what upstream uses. Their README says "GPL" (unversioned), but points to a COPYING file which is GPLv2. And the GPLv2 text explains that

> If the Program does not specify a version number of
this License, you may choose any version ever published by the Free Software
Foundation.

So on one hand they don't specify the license version, which implies gpl2Plus, but on the other they refer to the GPLv2 text, which implies gpl2Only.

Reply:
I usually check what other distributions do when uncertain about it: https://packages.fedoraproject.org/pkgs/chrpath/chrpath/

Reply:
I asked upstream: https://codeberg.org/pere/chrpath/issues/1

Reply:
Yes, it's GPL-2.0-or-later. I updated the expression.